Configuration/Settings.PunktDe.yaml

```diff
@@ -4,12 +4,22 @@ PunktDe:
       grantTypes:
         client_credentials:
           enabled: false
+          accessTokenTTL: PT1H
```

Comment:
Default values are set twice (1. in YAML configuration 2. in code with null coalescing operator).